samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -118,13 +120,26 @@ static void get_data(struct k_work *work)
 			&sensor_cli, NULL, &bt_mesh_sensor_time_since_presence_detected,
 			NULL);
 		if (err) {
-			printk("Error getting time since presence detected (%d)\n",
-			       err);
+			printk("Error getting time since presence detected (%d)\n", err);
 		}
 		break;
 	}
+	case (3): {
+		err = bt_mesh_sensor_cli_get(
+			&sensor_cli, NULL, &bt_mesh_sensor_present_amb_light_level, NULL);
+		if (err) {
+			printk("Error getting ambient light level (%d)\n", err);
+		}
+		break;
+	}
+	}
+
+	if (sensor_idx % 4) {
+		k_work_schedule(&get_data_work, K_MSEC(GET_DATA_INTERVAL_QUICK));
```

Comment:
I wonder something. The sensor index 1, 2, 3 will have quick "get" timeout, and then sensor index 0 will have slow timeout. This behavior is not mentioned anywhere in the updated docs and any debug text suggesting this is also not being printed (something like: "Getting data for 0xWXYZ property in 0xYX seconds"). I think this will create confusion when sample is used.

Can you just make all them have the same interval like before?

Reply:
It will be very hard to use the sample then. The reason when using the shell it is hard to use it when this is printed all the time. So that why get a quick run for all the values then wait before you get another round of all the values.